assets/src/components/account/UserProfile.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import {Box, Flex} from 'theme-ui';
+import {Button, Input, Paragraph, Title} from '../common';
+import * as API from '../../api';
+
+type Props = {};
+type State = {
+  fullName: string;
+  displayName: string;
+  profilePhotoUrl: string;
+  isLoading: boolean;
+  isEditing: boolean;
+};
+
+class UserProfile extends React.Component<Props, State> {
+  input: any = null;
+
+  state: State = {
+    fullName: '',
+    displayName: '',
+    profilePhotoUrl: '',
+    isLoading: true,
+    isEditing: false,
+  };
+
+  async componentDidMount() {
+    await this.fetchLatestProfile();
+
+    this.setState({isLoading: false});
+  }
+
+  fetchLatestProfile = async () => {
+    const profile = await API.fetchUserProfile();
+
+    if (profile) {
+      const {
+        display_name: displayName,
+        full_name: fullName,
+        profile_photo_url: profilePhotoUrl,
+      } = profile;
+
+      this.setState({
+        displayName,
+        fullName,
+        profilePhotoUrl,
+      });
+    } else {
```

Comment:
this is for a slight edge case in this method https://github.com/papercups-io/papercups/pull/66/files#diff-f310b3aeb13547d64314e5e0a19aa695R68-R72

where:
- someone hasn't created their profile yet
- they start entering info for their profile
- instead of clicking "save", they click "cancel"
- we reset the data to what's in the db, but since no record exists we just reset everything to blank